Dockerfile.rhel

```diff
@@ -8,31 +8,20 @@ ENV GIT_COMMITTER_NAME devtools
 ENV GIT_COMMITTER_EMAIL devtools@redhat.com
 LABEL com.redhat.delivery.appregistry=true
 
-WORKDIR /go/src/github.com/redhat-developer/service-binding-operator
-
-# Copy only relevant things (instead of all) to speed-up the build.
-COPY assets assets
-COPY build build
-COPY cmd cmd
-COPY deploy deploy
-COPY hack hack
-COPY manifests manifests
-COPY pkg pkg
-COPY test test
-COPY vendor vendor
-COPY go.mod .
-COPY go.sum .
-COPY LICENSE .
-COPY Makefile .
-COPY tools.go .
+# GOPATH-ish dir structure not needed anymore
+WORKDIR /tmp/service-binding-operator
```

Comment:
We can avoid repeating the path by declaring an argument before first `FROM` statement and then use it in all stages:

```
ARG BUILD_DIR=/tmp/service-binding-operator
.
.
FROM  openshift/origin-release:golang-1.13 AS builder
.
.
WORKDIR $BUILD_DIR
.
.
COPY --from=builder $BUILD_DIR/out/operator /usr/local/bin/service-binding-operator

Reply:
Done, thanks!

Reply:
Predrag, doesn't work 🤔  https://quay.io/repository/redhat-developer/app-binding-operator/build/01c0a07e-9645-4fd2-88ae-244933963252

Reply:
I am not authorized to view the logs, but I can confirm the issue locally, and was able to fix it. You need to redeclare the arg in each stage, as described in the [doc](https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact). Check this diff:

```diff
diff --git a/Dockerfile.rhel b/Dockerfile.rhel
index 33e2bf9..8f458e2 100644
--- a/Dockerfile.rhel
+++ b/Dockerfile.rhel
@@ -5,6 +5,8 @@ ARG BUILD_DIR=/tmp/service-binding-operator
 
 FROM openshift/origin-release:golang-1.13 AS builder
 
+ARG BUILD_DIR
+
 ENV LANG=en_US.utf8
 ENV GIT_COMMITTER_NAME devtools
 ENV GIT_COMMITTER_EMAIL devtools@redhat.com
@@ -27,6 +29,8 @@ RUN make build
 
 FROM registry.access.redhat.com/ubi7/ubi-minimal
 
+ARG BUILD_DIR
+
 LABEL com.redhat.delivery.appregistry=true
 LABEL maintainer "Devtools <devtools@redhat.com>"
 LABEL author "Shoubhik Bose <shbose@redhat.com>"
```

Reply:
Let me try - my local docker is defunct